lib/ast-converter.js

```diff
@@ -256,6 +256,26 @@ function fixExports(node, result, ast) {
 }
 
 /**
+ * Returns true if a given TSNode is a token
+ * @param  {TSNode} node the TSNode
+ * @returns {boolean}     is a token
+ */
+function isToken(node) {
+     /**
+     * Note: We treat JsxText like a token (TypeScript doesn't classify it as one),
```

Comment:
This sounds like a bug on us. I vaguely recall trying to fix this months ago but it was not a simple fix.